01-js-intro-and-jquery/js/javascript.js

```diff
@@ -0,0 +1,84 @@
+
+
+var functionFadeToggleSection = (function() {
```

Comment:
**Minor:** to link with [my previous comment](https://github.com/Demian-Moschin/startup/pull/2/files#r39558303), there's no need to use the word *"function"* here, since the content it's pretty self explanatory (it includes an explicit ``function`` definition on it).
Also always add an space after ``function``.